docs/app/Examples/views/Comment/Content/CommentExampleReplyForm.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { Button, Comment, Form, Icon } from 'stardust'
+
+const CommentExampleReplyForm = () => (
+  <Comment.Group>
+    <Comment>
+      <Comment.Avatar as='a' src='http://semantic-ui.com/images/avatar/small/steve.jpg' />
+      <Comment.Content>
+        <Comment.Author as='a'>Steve Jobes</Comment.Author>
+        <Comment.Metadata>
+          <div>2 days ago</div>
+        </Comment.Metadata>
+        <Comment.Text>Revolutionary!</Comment.Text>
+        <Comment.Actions>
+          <Comment.Action active>Reply</Comment.Action>
+        </Comment.Actions>
+        <Comment.ReplyForm>
```

Comment:
I'm really torn on this one.  While I agree entirely on the need for consistency with List.Icon, at the same time I really want to avoid this API:

```
<Comment.ReplyForm>
  <Form.TextArea />
  <Button labeled icon primary>
    <Icon name='edit' />
    Add Reply
  </Button>
</Comment.ReplyForm>
```

I think it comes off very confusing.  It would be great to get a single top level interface to creating a Comment form some how.